pkg/karmadactl/util/testing/fake.go

```diff
@@ -0,0 +1,35 @@
+package testing
+
+import (
+	cmdtesting "k8s.io/kubectl/pkg/cmd/testing"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+
+	"github.com/karmada-io/karmada/pkg/generated/clientset/versioned"
+	"github.com/karmada-io/karmada/pkg/karmadactl/util"
+)
+
+// TestFactory extends cmdutil.Factory
+type TestFactory struct {
+	*cmdtesting.TestFactory
+}
+
+var _ util.Factory = (*TestFactory)(nil)
+
+// NewTestFactory returns an initialized TestFactory instance
+func NewTestFactory() *TestFactory {
+	return &TestFactory{
+		TestFactory: cmdtesting.NewTestFactory(),
+	}
+}
+
+// KarmadaClientSet returns a karmada clientset
+func (t *TestFactory) KarmadaClientSet() (versioned.Interface, error) {
+	// TODO implement me
+	panic("implement me")
```

Comment:
Is it necessary to implement this method in the future?

Reply:
It's necessary in other commands test, e.g. `get`